chains/solana/contracts/programs/lockrelease-token-pool/src/context.rs

```diff
@@ -377,6 +377,18 @@ pub struct SetChainRateLimit<'info> {
     pub authority: Signer<'info>,
 }
 
+#[derive(Accounts)]
+#[instruction(mint: Pubkey)]
+pub struct SetRateLimitAdmin<'info> {
+    #[account(
+        seeds = [POOL_STATE_SEED, mint.key().as_ref()],
+        bump,
+    )]
+    pub state: Account<'info, State>,
```

Comment:
Looks like this should be `mut`, otherwise changes just won't be persisted, right?